src/frisby/spec.js

```diff
@@ -128,20 +130,29 @@ class FrisbySpec {
     this._fetch = fetch(this._request, { timeout: this.timeout() }) // 'timeout' is a node-fetch option
       .then(response => {
         this._response = new FrisbyResponse(response);
-        return response.textConverted();
-      })
-      .then(responseBody => {
-        let response = this._response;
-        response._body = responseBody;
-        // Auto-parse JSON
-        if (response.headers.has('Content-Type') && response.headers.get('Content-Type').includes('json') && response.status !== 204 && responseBody.length > 0) {
-          try {
-            response._json = JSON.parse(responseBody);
-          } catch(e) {
-            return Promise.reject(new TypeError(`Invalid json response body: '${responseBody}' at ${this._request.url} reason: '${e.message}'`));
-          }
+        if (this._setupDefaults.request && this._setupDefaults.request.rawBody) {
+          return response.arrayBuffer()
+            .then(buffer => {
+              this._response._body = buffer;
+            });
+        } else {
```

Comment:
Probably don't need the `else` here - this will be the default response to send when there are no special conditions.